WordPress/WordPressTest/MediaProgressCoordinatorTests.swift

```diff
@@ -96,4 +111,64 @@ class MediaProgressCoordinatorTests: XCTestCase {
         XCTAssertFalse(mediaProgressCoordinator.isRunning, "Coordinator should be stopped")
         XCTAssertFalse(mediaProgressCoordinator.hasFailedMedia, "Coordinator should not have failed media")
     }
+
+    func testAddToMediaProgress() {
+        let totalItems = 10
+
+        mediaProgressCoordinator.track(numberOfItems: totalItems)
+        XCTAssertNotNil(mediaProgressCoordinator.mediaGlobalProgress, "Progress should exist")
+
+        XCTAssertTrue(mediaProgressCoordinator.mediaGlobalProgress!.totalUnitCount == Int64(totalItems), "There should a total number of 10 items")
+
+        XCTAssertTrue(mediaProgressCoordinator.isRunning)
+
+        for index in 1...totalItems {
+            let progress = Progress.discreteProgress(totalUnitCount: 1)
+            mediaProgressCoordinator.track(progress: progress, ofObject: self.newTestMedia(), withMediaID: "/(index)")
```

Comment:
I don't think it actually affects these tests, but Swift string interpolation uses `\` instead of `/` – so all of these items will have the ID `/(index)`, not the actual index :)

Same for the next two tests too